src/api/integrations/managed-integrations.md

```diff
@@ -28,7 +28,7 @@ A Managed Integration is an [Integration][]  which a third party can control.
 | id                 | String                 | The Managed Integration's unique identifier.                                                                    |
 | type               | String                 | The type of Managed Integration. Supported types are `kete-enrolment`.                                          |
 | managedByAccountId | String                 | The id of the account managing the Managed Integration                                                          |
-| active             | Boolean                | Flag indicating whether the Managed Integration is active.                                                      |
+| enabled            | Boolean                | Flag indicating whether the Managed Integration is active.                                                      |
```

Comment:
Should the description say: 'Flag indicating whether the Managed Integration is **active**.'?